engine/src/core/input/input_manager.h

```diff
@@ -56,183 +58,166 @@ namespace Bald::Input {
-         * @brief                function that checks if key was pressed
+         * @fn IsKeyPressed
+         * @brief function that checks if key was pressed
          * @param keycode [int]  GLFW macro e.g. GLfW_KEY_SPACE
-         * @return        [bool] true if key was pressed otherwise false
+         * @return [bool] true if key was pressed otherwise false
          */
 
         [[nodiscard]] inline static bool IsKeyPressed(unsigned keycode) noexcept;
 
         /**
-         * @fn                   IsKeyTyped
-         * @brief                function that checks if key was typed
+         * @fn IsKeyTyped
+         * @brief function that checks if key was typed
          * @param keycode [int]  GLFW macro e.g. GLfW_KEY_SPACE
-         * @return        [bool] true if key was typed otherwise false
+         * @return [bool] true if key was typed otherwise false
          */
 
         [[nodiscard]] inline static bool IsKeyTyped(unsigned keycode) noexcept;
 
         /**
-         * @fn                      IsMouseButtonPressed
-         * @brief                   function that checks if mouse button was pressed
+         * @fn IsMouseButtonPressed
+         * @brief function that checks if mouse button was pressed
          * @param buttoncode [int]  GLFW macro e.g. GLFW_MOUSE_BUTTON_1
-         * @return           [bool] true if button was pressed otherwise false
+         * @return [bool] true if button was pressed otherwise false
          */
 
         [[nodiscard]] inline static bool IsMouseButtonPressed(unsigned buttoncode) noexcept;
 
         /**
-         * @fn                      IsMouseButtonTyped
-         * @brief                   function that checks if mouse button was typed
+         * @fn IsMouseButtonTyped
+         * @brief function that checks if mouse button was typed
          * @param buttoncode [int]  GLFW macro e.g. GLFW_MOUSE_BUTTON_1
-         * @return           [bool] true if button was typed otherwise false
+         * @return [bool] true if button was typed otherwise false
          */
 
         [[nodiscard]] inline static bool IsMouseButtonTyped(unsigned buttoncode) noexcept;
 
         /**
-         * @fn                  GetMousePos
-         * @brief               sets two variables on current mouse position
+         * @fn GetMousePos
+         * @brief sets two variables on current mouse position
          * @param xpos [double] x-coordinate
          * @param ypos [double] y-coordinate
          */
 
         inline static void GetMousePos(double& xpos, double& ypos) noexcept;
 
         /**
-         * @fn                  SetKeyPressedCallback
-         * @brief               sets function which will be called on key pressed event
-         * @param fun           function pointer
-         * @param args          arguments for function
+         * @fn SetKeyPressedCallback
+         * @brief sets function which will be called on key pressed event
+         * @param [F&&] fun -> function pointer
```

Comment:
probably should mansion that function has to take Evnet& as the first parameter.